src/containers/Trips/MulticityTrip.js

```diff
@@ -0,0 +1,105 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Redirect } from 'react-router-dom';
+import MulticityTripView from '../../views/trips/Multicity';
+import Submit from '../../components/SubmitButton';
+import Dashboard from '../../views/Dashboard/sidebar/index';
+import TripType from '../../components/trips/TripTypes';
+import TripHeader from '../../components/trips/TripHeader';
+import { getCitiesAction } from '../../redux/actions/createAccommodationActions';
+import { addMoreTrip,
+  removeMoreTrip,
+  editMultipleTrip, 
+  sendMulticityTrip } from '../../redux/actions/multicityActions';
+import Spinner from '../../components/Spinner';
+import { updateSpinnerStatus } from  '../../redux/actions/profileActions';
+
+export class MulticityTrip extends Component {
+
+   componentDidMount() {
+    const { getCitiesAction } = this.props;
+     getCitiesAction();
+  }
+
+  tripActionsProps = (index,item ) => {
+    return {
+      cities: this.props.cities,
+      tripIndex: index,
+      removeTrip: this.onHandleRemoveMoreTrip,
+      onHandleInputChange: this.onHandleInputChange,
+      data: this.props.multicityTripData.data,
+      itemRequests: item,
+    };
+  }
+
+  onHandleAddMoreTrip = () => {
+    const { addMoreTrip } = this.props;
+    addMoreTrip();
+  }
+
+  onHandleRemoveMoreTrip = (item) => {
+    const { removeMoreTrip } = this.props;
+    removeMoreTrip(item);
+  }
+
+  onHandleInputChange = (e, tripIndex) => {
+    const { name, value } = e.target;
+    const { editMultipleTrip } = this.props;
+    editMultipleTrip({ name, value, tripIndex });
+  }
+
+  onHandleSubmit = (e) => {
+    e.preventDefault();
+    const { sendMulticityTrip, multicityTripData, updateSpinnerStatus } = this.props;
+    updateSpinnerStatus(true);
+    sendMulticityTrip(multicityTripData);
+  }
+
+  render() {
+    const { multicityTripData, viewProfile } = this.props;
+    return (
+      <Dashboard>
+        {viewProfile.spinner && <Spinner/>}
+        <div className="oneway-container">
+          <TripHeader />
+          { multicityTripData.message && <Redirect to="/requests" /> }
```

Comment:
I prefer you redirect the user to specific request i.e
`<Redirect to="/requests/:tripRequestId" />  `